OsmAnd/src/net/osmand/plus/plugins/audionotes/AudioNotesLayer.java

```diff
@@ -231,11 +231,12 @@ public void getRecordingsFromPoint(PointF point, RotatedTileBox tileBox, List<?
 		int ey = (int) point.y;
 		int compare = getScaledTouchRadius(getApplication(), getRadiusPoi(tileBox));
 		int radius = compare * 3 / 2;
-		for (Recording n : plugin.getAllRecordings()) {
-			PointF pixel = NativeUtilities.getPixelFromLatLon(getMapRenderer(), tileBox, n.getLatitude(), n.getLongitude());
+		for (Recording recording : plugin.getAllRecordings()) {
+			PointF pixel = NativeUtilities.getElevatedPixelFromLatLon(getMapRenderer(), tileBox,
```

Comment:
Would be better to calculate latlon bbox like in MapillaryVectorLayer.java before for loop.
Implement such approach in all similar places.